packages/babel-helper-module-transforms/src/rewrite-this.js

```diff
@@ -13,9 +14,10 @@ const rewriteThisVisitor = {
   },
   Function(path) {
     if (!path.isArrowFunctionExpression()) path.skip();
+    if (path.isMethod()) skipAllButComputedKey(path);
```

Comment:
A method is never an arrow function, so it will always be skipped by the line before.
We should do
1. If it is a Method,
  i. skipAllButComputedKey
2. Else, if it is not an arrow,
  i. skip